### PR TITLE
Change link text after triggering.

### PR DESCRIPTION
**Pull request by HelenaEksler**
#215

### CODE REVIEW

**Comment by HelenaEksler** [comment]
shoov-trigger1

File: shoov/modules/custom/shoov_ci_build/shoov_ci_build.module [hunk 1]
```diff
@@ -45,10 +45,16 @@ function  shoov_ci_build_trigger_ci_build_item($message, $ajax = 'nojs') {
   );
   $url = implode('/', $url);
   drupal_http_request($url);
+
+  $commands = array();
```
Comment (amitaibu) [inline shoov/modules/custom/shoov_ci_build/shoov_ci_build.module]: move below the `if (!$ajax) {`

**Comment by amitaibu** [comment]
:+1:

**Comment by amitaibu** [comment]
Merged, thanks.

**Pull request closed without merging**